libraries/CurieSoftwareSerial/SoftwareSerial.h

```diff
@@ -66,7 +67,6 @@ class SoftwareSerial : public Stream
   void tx_pin_write(uint32_t pin_state) __attribute__((__always_inline__));
   void setTX(uint8_t transmitPin);
   void setRX(uint8_t receivePin);
-  void setRxIntMsk(bool enable);
```

Comment:
Is this function no longer necessary? Maybe it should get removed from .cpp file as well?